src/guiapi/src/screen_unloop.cpp

```diff
@@ -0,0 +1,64 @@
+// screen_unloop.cpp
+//unlooping screens via C++ with C interface
+
+#include "screen_unloop.h"
+#include "gui.h"
+#include "screens.h"
+
+//I could not use functions to initialize those arrays in C
```

Comment:
Yes, that is the side effect of wrapping the screen pointers in `get_scr_` functions. All statically initialized data using screen pointers are now going to have to be initialized at runtime :(

Reply:
original `extern screen_t*` behaved exactly same

Reply:
> All statically initialized data using screen pointers are now going to have to be initialized at runtime :(
I want to rewrite all screens to Cpp, so I will be able to use constexpr